packages/stex-react-renderer/src/lib/SubProblemAnswer.tsx

```diff
@@ -200,14 +200,15 @@ export function SubProblemAnswer({
           border: `1px solid ${PRIMARY_COL}`,
         }}
       >
-        {mmtHTMLToReact(subProblem.solution)}
+        {mmtHTMLToReact(subProblem.solution.replace(/__mmt/g,''))}
```

Comment:
Export `MMT_CUSTOM_ID_PREFIX` and use that instead of `_mmt`. Also, note in a comment that the custom tag was added to the subProblem node to display the SubProblemAnswer in-place which is unfortunately the solution node as well.